rero_ils/modules/entities/sync.py

```diff
@@ -325,48 +339,54 @@ def sync_record(self, pid):
         updated = error = False
         try:
             # get contribution in db
-            agent = Entity.get_record_by_pid(pid)
-            if not agent:
+            entity = Entity.get_record_by_pid(pid)
+            if not entity:
                 raise Exception(f'ERROR MEF {pid} does not exists in db.')
             self.logger.debug(f'Processing MEF(pid: {pid})')
-            # iterate over all agent sources: rero, gnd, idref
-            doc_pids = self._get_documents_pids_from_mef(agent.pid)
+            # iterate over all entity sources: rero, gnd, idref
```

Comment:
Maybe for performance, if entity type is not bf:Person or bf:Organisation, you don't need to check gnd and rero, only idref? But I don't know if it's worth differentiating the behaviour.